pkg/cmd/server/admin/overwrite_bootstrappolicy.go

```diff
@@ -173,14 +186,23 @@ func OverwriteBootstrapPolicy(optsGetter restoptions.Getter, policyFile, createB
 		}
 		runtime.DecodeList(template.Objects, kapi.Codecs.UniversalDecoder())
 
+		errs := []error{}
 		for _, item := range template.Objects {
 			switch t := item.(type) {
 			case *authorizationapi.Role:
 				ctx := kapi.WithNamespace(kapi.NewContext(), t.Namespace)
 				if change {
 					roleStorage.Delete(ctx, t.Name, nil)
-					if _, err := roleStorage.CreateRoleWithEscalation(ctx, t); err != nil {
+					if err := kclient.RetryOnConflict(tryTryAgainBackoff, func() error {
```

Comment:
> it is weird to retryonconflict from a create call

No, I mean in `pkg/authorization/registry/role/policybased/virtual_storage.go`, detecting a conflict during the policy update and retrying the role update/create/delete in that code.